### DISCUSSION TITLE
Allowing ^2.0 phpspreadsheet versions

### DISCUSSION THREAD

**Discussion by TurboThomsen** [solspace/craft-freeform#1643] [Feature Requests]
Hi

I wanted to ask, if it is possible to allow ^2.0 phpspreadsheet versions for craft 4 and 5?
Currently I am updating a site from Craft 3 to Craft 4 with multiple plugins requirering the spreadsheet ^2.0 version and I cant install different versions at once...

Thanks

**Comment by kjmartens** [comment]
Sure thing! I've added it now and it'll be in the next release 🙂

**Comment by TurboThomsen** [comment]
Can you pls add it to v4 also? 😅

**Reply by kjmartens** [reply]
Yes, this is now available in Freeform 4.1.23 🙂
